### ISSUE TITLE
Multiline edit with text wrap produces odd results

### ISSUE THREAD

**Issue by onomou** [notepad-plus-plus/notepad-plus-plus#8026]
### Description of the Issue

With text wrap enabled, a multiline edit going past the end of the (soft-wrapped) line causes unpredictable cursor behavior and results in the cursor returning to the beginning of the same line (and possibly another line also).
### Steps to Reproduce the Issue

1. Fill two lines with text
2. Turn on text wrapping
3. Place the cursor at the end of the first line and activate multiline editing with Ctrl + Shift + Down to place the cursor at the end of each line
4. Type until the cursor runs past the end

### Expected Behavior

The multiline cursors should wrap properly to the next (soft-wrapped) line and continue to allow text entry
### Actual Behavior

The multiline cursors returned to the beginning of the original line plus one character, and text entry was put there.
### Debug Information

```
Notepad++ v7.7.1   (32-bit)
Build time : Jun 16 2019 - 21:14:50
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS : Windows 8.1 (64-bit)
Plugins : ComparePlugin.dll DSpellCheck.dll mimeTools.dll NppConverter.dll nppcrypt.dll NppExport.dll NppTaskList.dll NppTextFX.dll SecurePad.dll 
```

![npp-multiselection-wrap](https://user-images.githubusercontent.com/131693/76238105-4f04ee00-626a-11ea-8fca-6d406c02c0ed.gif)

**Comment by rddim** [comment]
Confirm, this exist in v7.7.1, but does **not** exist in latest v7.8.5

**Comment by sasumner** [comment]
Since @rddim says it doesn't exist in 7.8.5, I'll ask @onomou to retest using 7.8.5.

**Comment by onomou** [comment]
I should have updated first (oops), but it still is broken in 7.8.5. See gif here:
![npp-multiselection-wrap-again](https://user-images.githubusercontent.com/131693/76269518-f3a81f80-62ac-11ea-9940-74a71979cfe2.gif)


```
Notepad++ v7.8.5   (32-bit)
Build time : Mar  4 2020 - 11:04:20
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS Name : Windows 8.1 Enterprise (64-bit) 
OS Build : 9600.0
Plugins : none
```

**Comment by rddim** [comment]
Strange! Last night when I test it at home (Win 7) I have no problems. Today I test it at office PC with Win 10 and this issue appear. Now I make the same test at home again (Win 7) and this issue appear.

Tested with both latest x86 and x64 versions.

**Comment by sasumner** [comment]
I agree that this is exhibiting unexpected behavior (on 7.8.5) as originally reported by @onomou with a couple of minor comments on the original posting:

* it's Shift+Alt + Down (not Shift + Ctrl + Down) 
* for me, it put the carets in the 3rd column of the lines after the lines wrapped, not the 2nd column
